qa/integration-tests/src/test/java/io/camunda/zeebe/broker/it/network/SmallMessageSizeTest.java

```diff
@@ -27,78 +28,72 @@
   private static final DataSize MAX_MESSAGE_SIZE = DataSize.ofKilobytes(12);
   private static final String LARGE_TEXT =
       "x".repeat((int) (MAX_MESSAGE_SIZE.toBytes() / VARIABLE_COUNT));
+  private static final String JOB_TYPE = "acme";
 
   private static final EmbeddedBrokerRule BROKER_RULE =
-      new EmbeddedBrokerRule(b -> b.getNetwork().setMaxMessageSize(MAX_MESSAGE_SIZE));
+      new EmbeddedBrokerRule(
```

Comment:
This issue was already in the existing implementation, and not introduced by this PR. Nevertheless, I would like to discuss it:

There is a cardinality mismatch between the rule, which is for the whole class and the before method, which is per test case.

IMHO this makes the test hard to understand and maintain, because state is accumulated between test methods. For each test method a new, but structurally identical process is deployed. And then a new process instance is started. But each test is written as if there would be only one job instance that can be activated. 

So either the rule could be non-static, which slows down the test. Or a new task type could be generated for each test method (e.g. `UUID.randomUUIID().toString`)